docs/getting_started_guide/Installation_Options/ISO/ISO_Build/Environment_Preparation.rst

```diff
@@ -128,8 +128,25 @@ Start the docker daemon:
   $ sudo systemctl start docker
   $ sudo systemctl enable docker
 
+Build Your Build Containers
+---------------------------
+
+The `simp-core`_ project provides suitable build Dockerfiles for both EL6 and
+EL7 in the ``build/Dockerfiles`` directory.
+
+These work well for building both CentOS 6 and 7 artifacts and the usage is
+noted at the top of those files.
+
+Unfortunately, getting this to work with RHEL has proven to be a challlenge so
```

Comment:
'challlenge' -> 'challenge'

Reply:
The section is 'building your build containers', but there are no instructions on how to build the containers.  Also, how this relates the beaker:suites[rpm_docker] is a bit confusing.

Reply:
@lnemsick-simp It doesn't. That's an environment prep file so you need to do those things to prep your environment. While it needs to be updated at some point, it adds information that was previously missing that should help RHEL users know how to prep their box and be able to build the rest of the system.